src/azul/plugins/metadata/anvil/service/response.py

```diff
@@ -203,34 +203,31 @@ def _non_pivotal_fields_by_entity_type(self) -> dict[str, set[str]]:
             },
             'biosamples': {
                 'anatomical_site',
+                'apriori_cell_type',
                 'biosample_type',
                 'disease',
                 'donor_age_at_collection_unit',
                 'donor_age_at_collection',
             },
             'datasets': {
+                'consent_group',
+                'data_modality',
+                'data_use_permission',
                 'dataset_id',
                 'duos_id',
+                'owner',
+                'principal_investigator',
+                'registered_identifier',
                 'title'
             },
             'diagnoses': {
                 'disease',
                 'phenotype',
                 'phenopacket',
+                'onset_age',
                 'onset_age_unit',
+                'diagnosis_age',
                 'diagnosis_age_unit',
-                *(
-                    # These fields are of high cardinality, but the number of
-                    # aggregated inner entities per donor should be low. Since
-                    # diagnoses do not appear in the index as outer entities,
-                    # this is our only opportunity to display these fields.
-                    [
-                        'diagnosis_age',
-                        'onset_age'
-                    ]
-                    if self.entity_type == 'donors' else
-                    []
-                )
```

Comment:
The aggregations for these fields may be very large in dataset inner entities. Are we sure we want to include them?